publication-commons/src/main/java/no/unit/nva/publication/model/business/importcandidate/Imported.java

```diff
@@ -0,0 +1,74 @@
+package no.unit.nva.publication.model.business.importcandidate;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.model.Username;
+import nva.commons.core.JacocoGenerated;
+
+import java.net.URI;
+import java.time.Instant;
+import java.util.Objects;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class Imported implements ImportStatus {
+
+    private final static String IMPORTED_DATE = "importedDate";
+    private final static String PUBLICATION_ID = "publicationId";
+    private final static String IMPORTED_BY = "importedBy";
+
+    @JsonProperty(IMPORTED_DATE)
+    private final Instant importedDate;
+
+    @JsonProperty(PUBLICATION_ID)
+    private final URI publicationId;
+
+    @JsonProperty(IMPORTED_BY)
+    private final Username importedBy;
+
+    public Imported(@JsonProperty(IMPORTED_DATE) Instant importedDate,
```

Comment:
Mangler kanskje @JsonCreator over konstruktøren

Reply:
Is importedDate when something happened outside the datamodel?
if not use created/modified (when this entity changed inside the model).
Since the class is named imported, you don't need to repeat it in the fields as well.

Reply:
ImportedDate =/= created date. 
I think created date is the date of when the entity was created. (started to exist in database).
An importCandidate might have been created months ago by ScopusHandlerBot before it gets imported by a human to NVA resource database.